code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -355,8 +370,8 @@
 			S.start()
 		qdel(B)
 
-/datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
-	name = "!(%@ ERROR )!@$"
+datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
+	name = "Odd Black Liquid"
```

Comment:
Yes but you still left it an option, I'm saying no, don't do that ever stop what are you doing no pls.